toonz/sources/toonz/sceneviewerevents.cpp

```diff
@@ -1024,7 +1024,7 @@ bool SceneViewer::event(QEvent *e) {
   if (e->type() == QEvent::MouseButtonPress)
     clock.start();
   else if (e->type() == QEvent::MouseMove) {
-    if (clock.elapsed() < 10) {
+    if (clock.isValid() && clock.elapsed() < 10) {
```

Comment:
@manongjohn 

> Regarding the change you put in to fix the circle stuck at center at first launch, will that fix all the tools as well or only the brush?

The fix was done in here in `SceneViewer::event()` so yes, it will fix not only the brush tool but all the tools.
Thank you for letting know that. I'll update my comment.